packages/styled-components/src/constructors/test/keyframes.test.js

```diff
@@ -2,11 +2,12 @@
 import React from 'react';
 import TestRenderer from 'react-test-renderer';
 
-import css from '../css';
-import keyframes from '../keyframes';
-import Keyframes from '../../models/Keyframes';
 import { expectCSSMatches, getCSS, resetStyled } from '../../test/utils';
```

Comment:
the order definitely matters here. And I feel the mock call should always come first.